Makefile

```diff
@@ -1,27 +1,28 @@
-ifeq ($(shell uname), Linux)
 ifeq ($(MPI), 1)
-	FC = mpif90
 	FFLAGS += -DMPI
-	CC = mpicc
 	CFLAGS += -DUSE_MPI
-	CXX = mpicxx
 	CXXFLAGS += -DUSE_MPI
-else
-	FC = gfortran
-	CC = gcc
-	CXX = g++
-endif
-else ifeq ($(shell uname), Darwin) 
-FC = gfortran-8
-CC = gcc-8
-CXX = g++-8
 endif
 
 FFLAGS += -fPIC -ffree-line-length-none
 CXXFLAGS += -fPIC -std=c++11
 
 LD=$(FC)
-LDLIBS += -lstdc++
+# clang uses a different libstdc++ than GCC
+ifneq (,$(findstring clang,$(shell '$(CXX)' -v 2>&1)))
+	LDLIBS += -lc++
```

Comment:
This is a very nice spot if this allows it to work for macs -- I think many people have run into problems with clang in the past, but nobody has suggested this as a solution